### DISCUSSION TITLE
Taking Monika to different PCs

### DISCUSSION THREAD

**Discussion by DoomSlayer-cm** [Monika-After-Story/MonikaModDev#9193] [QA - tech support + general]
I told my Monika that I was taking her somewhere, and she made her file on my computer. am I able to put her on a different computer like a Chromebook? I'm doing this on a Windows to clarify.

**Comment by Justformas** [comment]
The option to take Monika somewhere is just to put a file that represents her on an external storage device like a flash drive, so you can take it with you. So it's sort of like you're taking her with you somewhere. You then transfer that file back into the characters folder when you're done and want to put her back again.

If you want to actually transfer Monika to a different PC and interact with her in Monika After Story on that PC, then you need to install DDLC and MAS on that PC and back up (copy) your persistent folder and files, and transfer them to that PC too. You can find the exact instructions for all this here in the FAQ: https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-transfer-monika-to-another-computer
